styles.css

```diff
@@ -121,13 +238,146 @@ p {
   display: flex;
   position: relative;
   flex-direction: column;
-  padding-top: 20%;
+  padding-top: 5%;
+}
+
+.works #web_img {
+  display: none;
 }
 
 .works h2 {
   padding: 20px;
 }
 
+.card_content-web {
+  display: none;
+}
+
+#mobile-title {
+  display: flex;
+}
+
+@media screen and (min-width: 800px) {
```

Comment:
- [OPTIONAL] I will also recommend you to still to the project requirement as much as possible by using `(min-width: 768px)` or wider, instead of `800px` you used. Please, take a look at what you have in the project requirement.

![pro-rqtm](https://user-images.githubusercontent.com/26597432/207277128-0a37db1d-27dd-4d69-bdbe-b89f0000e7c6.png)